src/routes/login/+page.svelte

```diff
@@ -143,24 +139,22 @@
             </label>
             <div class="relative">
                 <input
-                    class="bg-surface-700 text-white border rounded-full transition-colors {passwordIndicator} w-full focus:border-primary-500 focus:outline-none focus:ring-0"
+                    class="input {passwordIndicator}"
                     type={showPassword ? "text" : "password"}
                     id="password_input"
                     placeholder="Dein super sicheres Passwort 😉"
                     bind:this={passwordReference}
                 />
-                <div>
-                    <button
-                        class=" absolute bg-surface-700 w-14 border-surface-500 inset-y-0 h-8 top-1 right-1 border-l-2 border-surface-400 pl-4 flex items-center text-md rounded-r-full"
-                        on:click={() => (showPassword = !showPassword)}
-                    >
-                        {#if showPassword}
-                            {btnIcon.unlocked}
-                        {:else}
-                            {btnIcon.locked}
-                        {/if}
-                    </button>
-                </div>
+                <button
+                    class="text-md absolute inset-y-0 right-1 top-1 flex h-8 w-14 items-center rounded-r-full border-l-2 border-surface-500 bg-surface-700 pl-4"
```

Comment:
Hab ich vorher nicht gesehen, aber hier können noch ein paar CSS Klassen raus.
```suggestion
                    class="absolute right-1 top-1 flex h-8 w-14 items-center rounded-r-full border-l-2 border-surface-500 bg-surface-700 pl-4"
```